db/migrate/20150819105725_add_lookup_value_match_to_host_and_hostgroup.rb

```diff
@@ -5,10 +5,20 @@ def up
     Host::Managed.reset_column_information
     Hostgroup.reset_column_information
 
-    Host::Managed.where(:type => 'Host::Managed').find_in_batches(:batch_size => 100) do |group|
-      group.each do |host|
-        host.update_attribute(:lookup_value_matcher, host.send(:lookup_value_match))
-      end
+    matchers = LookupValue.where("lookup_values.match like ?", "fqdn=%").group("lookup_values.match").count.keys
+    fqdns = matchers.map{|m| m.gsub(/^fqdn=/, "")}
+    hostnames = fqdns.map{|fqdn| fqdn.split(".").first}
+    if SETTINGS[:unattended]
+      hosts = Host::Managed.where(:nics => {:name => (fqdns + hostnames)}).joins(:primary_interface)
```

Comment:
I don't like that syntax...
```ruby
foo = if bar
  baz
else
  barvaz
end
```

it just makes very little sense to me that an if statement returns a value.